logbook/more.py

```diff
@@ -268,7 +268,7 @@ def __init__(self, arguments, stdin_format=None,
         self._subprocess = subprocess
 
     def emit(self, record):
-        args = [arg.format(record=record).encode(self.encoding)
+        args = [arg.format(record=record)
```

Comment:
I'm not sure why the parameters were being encoded before, I can't see in the subprocess documentation that bytes are required.